public/static/docs/command-reference/diff.md

```diff
@@ -46,148 +58,139 @@ for example when `dvc init` was used with the `--no-scm` option.
 
 ## Examples
 
-For these examples we can use the chapters in our
-[Get Started](/doc/get-started) section, up to
-[Add Files](/doc/get-started/add-files).
+For these examples we can use the [Get Started](/doc/get-started) project.
 
 <details>
 
-### Click and expand to setup example
+### Click and expand to setup the project to run examples
 
-Start by cloning our example repo if you don't already have it. Then move into
-the repo and checkout the
-[3-add-file](https://github.com/iterative/example-get-started/releases/tag/3-add-file)
-tag, corresponding to the [Add Files](/doc/get-started/add-files) _Get Started_
-chapter:
+Start by cloning our example repo if you don't already have it:
 
 ```dvc
 $ git clone https://github.com/iterative/example-get-started
 $ cd example-get-started
-$ git checkout 3-add-file
 ```
 
-Download the precomputed data using:
+Download data using:
 
 ```dvc
-$ dvc pull
+$ dvc fetch -T
 Preparing to download data from 'https://remote.dvc.org/get-started'
 ...
 ```
 
-</details>
+The `-T` flag passed to `dvc fetch` makes sure we have all the data files
+related to all existing tags in the repo. You may see the available tags of our
+example repo [here](https://github.com/iterative/example-get-started/tags).
 
-## Example: Previous commit in the same branch
+</details>
 
-The minimal `dvc diff`, run without arguments, defaults to comparing DVC-tacked
-files between `HEAD` (current Git commit) and the current <abbr>workspace</abbr>
-(uncommitted changes, if any).
+## Example: Checking workspace changes
 
-To see the difference between the very previous commit of the project and the
-workspace, we can use `HEAD^` as `a_ref`:
+The minimal `dvc diff`, run without arguments, defaults to comparing DVC-tracked
+files between `HEAD` (last Git commit) and the current <abbr>workspace</abbr>
+(uncommitted changes, if any):
```

Comment:
One more thing, actually!

> current workspace (uncommitted changes

So. I thought about this again and I think that we could make the distinction between this command and `dvc status` clearer by calling them "changes in the workspace (not yet tracked by DVC)" in status, and "current workspace (tracked but uncommitted changes)" here in diff. It's not the simplest language but at least it's a first attempt at clarifying this discrepancy between commands and their purpose.

What do you think Ivan?

p.s. I'm already updating `dvc status` along these lines in PR #1014

Reply:
it's super confusing. As well as using "uncommitted changes, if any". Not sure that `changes in the workspace (not yet tracked by DVC)` is correct, also not sure this the right place to discuss `dvc status` that probably should be reviewed holistically.

You are right that this phrase should changed. I would wait for the discussion in one of the `dvc diff` tickets to complete. It'll be the next iteration of this doc.